composer.json

```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
+        "ext-mongo": "mongodb extetension to use MongoDB writer",
         "zendframework/zend-escaper": "Zend\\Escaper component, for use in the XML formatter",
-        "zendframework/zend-mail": "Zend\\Mail component",
-        "zendframework/zend-validator": "Zend\\Validator component"
+        "zendframework/zend-mail": "Zend\\Mail component to use Email writer",
+        "zendframework/zend-validator": "Zend\\Validator component to use filter feature"
```

Comment:
`Zend\Validator` is for using validators, not filters, so shouldn't the message be "to use validator feature"? [`zend-filter`](https://github.com/zendframework/zend-filter) is another component.